recipe/meta.yaml

```diff
@@ -25,25 +25,31 @@ requirements:
     - python >=3.6
     # Required
     - affine >=2.2
+    - blas =*.*=mkl
```

Comment:
We intentionally left this out as suggested by the reviwers: https://github.com/conda-forge/staged-recipes/pull/9220#issuecomment-543148080

Reply:
Again -- should we then "backport" the removal of this requirement from meta.yaml to environment.yml? I don't see anything against it in the discussion...

Reply:
Me neither. If it unexpectedly breaks our builds, we revert this. Let's try it.